packages/manager/apps/web/client/app/hosting/database/order-private/translations/Messages_fr_FR.json

```
@@ -0,0 +1,47 @@
+{
+  "hosting_database_order_private_title": "Commander un SQL Privé",
+  "hosting_database_order_private_description": "Vous devez avoir un hébergement pour commander les options SQL Privé et Start SQL.",
+  "hosting_database_order_private_step_header_type": "Type de base de données",
+  "hosting_database_order_private_step_header_customize": "Personnalisation",
+  "hosting_database_order_private_step_header_duration": "Choisissez la durée",
+  "hosting_database_order_private_step_header_activation": "Activation",
+  "hosting_database_order_private_step_header_payment": "Paiement",
+  "hosting_database_order_private_step_description_activation": "Afin de confirmer votre demande, veuillez accepter les conditions d'utilisation.",
+  "hosting_database_order_private_step_description_payment": "Afin de confirmer votre demande, veuillez choisir votre moyen de paiement, et accepter les conditions d'utilisation.",
+  "hosting_database_order_private_submit_activate": "Activer",
+  "hosting_database_order_private_submit_pay": "Payer",
+  "hosting_database_order_private_cancel": "Annuler",
+  "hosting_database_order_private_select_placeholder": "Sélectionner...",
```

@FredericEspiau might say 😉 
```suggestion
  "hosting_database_order_private_select_placeholder": "Sélectionner…",
```